README.md

```diff
@@ -137,3 +137,30 @@ Open your browser and navigate to [http://localhost:4200](http://localhost:4200)
 ---
 
 [Details regarding architecture](ARCHITECTURE.md)
+
+
+# The Open Mina documentation overview
```

Comment:
Maybe move this to the top of the README? @tizoc any suggestions?

Reply:
Juraj wants it at the bottom of the readme